.gitignore

```diff
@@ -1,3 +1,4 @@
 /out
 /cache
 *.log
+.DS_Store
```

Comment:
I think we don't typically ignore this in projects.  Mac people might add it to a global ignore, you might check with someone who primarily uses a mac.

Also, naz had to catch this a bunch for me, but we keep new lines at the end of the files.  Your editor might be linting them out too.